newIDE/app/src/SceneEditor/Toolbar.js

```diff
@@ -140,12 +140,12 @@ export class Toolbar extends PureComponent<Props> {
             {
               label: 'Zoom in',
               click: this.props.zoomIn,
-              accelerator: 'CmdOrCtrl++',
+              accelerator: 'CmdOrCtrl+numadd',
```

Comment:
Is this working on both the web-app and the Electron app?

Reply:
Displaying accelerators isn't implemented for the web-app menus yet, as far as I can see [here](https://github.com/4ian/GDevelop/blob/a47acbb82a3c19ed8508f40be293e4eec2cb2139/newIDE/app/src/UI/Menu/MaterialUIMenuImplementation.js#L76-L79) in `MaterialUIMenuImplementation.js`. On the Electron app, the shortcuts are displayed as `Ctrl++` and `Ctrl+-` on Linux/Windows platforms. I don't know how they are displayed on MacOS platforms.

Reply:
Oops you're right about the web-app! :) 
Can you check if `CmdOrCtrl+Plus` is displayed correctly? If yes I would favor that.

Reply:
CmdOrCtrl+Plus is shown as `Ctrl+Shift+=` on Linux/Windows, since the `+` is up on the `=` key. I'm not sure about MacOS, but the screenshot in #1593 seems to indicate that its shown correctly as ⌘+. I'm not sure what to do here :sweat_smile: